hack/setup-kind-in-ci.sh

```diff
@@ -411,6 +416,7 @@ setup_kind_singlecluster() {
     --set kiali_internal.cache_expiration.istio_status="0" \
     --set kiali_internal.cache_expiration.mesh="10s" \
     --set kiali_internal.cache_expiration.waypoint="2m" \
+    --set kiali_internal.graph_cache.enabled="false" \
```

Comment:
If we are going to enable the graph cache by default for users we should have our test suite use it by default to catch any potential issues.